src/modules/SignUp/StepperLayout/index.module.scss

```diff
@@ -0,0 +1,43 @@
+@import 'src/styles/constants/_primitives.scss';
+
+.container {
+    display: flex;
+    width: 100%;
+    height: 100vh;
+}
+
+.vertical-stepper {
+    width: 440px;
+    min-width: 440px;
+    flex-direction: column;
+    gap: 80px;
+    padding: 2rem;
+    background-color: $color-grey-50;
+
+    &__footer {
+        margin-top: auto;
+        margin-bottom: 0;
+        display: flex;
+        justify-content: space-between;
+
+        &__text {
+            font-size: 0.875rem;
+            font-weight: 400;
+            line-height: 1.25rem;
+            color: $color-grey-600;
+        }
+    }
+}
+
+.form {
+    flex-grow: 1;
+    padding: 6rem 2rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+
+    @media (max-width: 640px) {
```

Comment:
we don't have `640px` in responsive sizes (talwind). you should use `$md` (768px)